include/Graphs/SVFG.h

```diff
@@ -394,43 +395,47 @@ class SVFG : public VFG
     }
 
     /// Add memory Function entry chi SVFG node
-    inline void addFormalINSVFGNode(const MemSSA::ENTRYCHI* chi)
+    inline void addFormalINSVFGNode(const FunEntryBlockNode* funEntry,  const MRVer* resVer)
     {
-        FormalINSVFGNode* sNode = new FormalINSVFGNode(totalVFGNode++,chi);
-        addSVFGNode(sNode, pag->getICFG()->getFunEntryBlockNode(chi->getFunction()));
-        setDef(chi->getResVer(),sNode);
-        funToFormalINMap[chi->getFunction()].set(sNode->getId());
+        FormalINSVFGNode* sNode = new FormalINSVFGNode(totalVFGNode++, resVer->getMR()->getPointsTo(), funEntry->getFun());
```

Comment:
We will need to be consistent for the arguments when creating `FormalOUTSVFGNode` and `FormalInSVFGNode`.  If the argument is a `const MRVer* `, we do not need to passing pointsTo since an MRVer carries the pointsTo as a field. Please check